### PR TITLE
gh-103363: Add follow_symlinks argument to `pathlib.Path.owner()` and `group()`

### PR DESCRIPTION
**Pull request by kamilturek**
* Issue: gh-103363

----
:books: Documentation preview :books:: https://cpython-previews--107962.org.readthedocs.build/

### CODE REVIEW

**Comment by AA-Turner** [comment]
(removing the *closing* keyword so as not to close the issue automatically/prematurely)

File: Lib/test/test_pathlib.py [hunk 1]
```diff
@@ -30,6 +30,20 @@ def test_is_notimplemented(self):
...
+    all_users = []
+
+
+try:
+    import grp
+    all_groups = [g.gr_gid for g in grp.getgrall()]
+except (ImportError, AttributeError):
+    all_groups = []
+
+
```
Comment (barneygale) [inline Lib/test/test_pathlib.py]: Could these be moved to a helper method, or inline in the tests that use em? They're not used unless the test suite is being run as root, which is a fairly rare configuration. You can call `self.skipTest()` to skip from the test method body.

**Review by barneygale — CHANGES_REQUESTED** [review]
Sorry it took me ages to review. Overall looks really good, thank you. Could you merge in the latest `main`? You'll find there's a new `_PathBase` class with its own `owner()` and `group()` methods - please add the *follow_symlinks* argument there too. Thanks again!

File: Lib/test/test_pathlib.py (hunk 1 above)
Comment (kamilturek) [reply Lib/test/test_pathlib.py]: Makes sense. Done 🙂

**Comment by kamilturek** [comment]
I have made the requested changes; please review again

**Comment by kamilturek** [comment]
I have made the requested changes; please review again

**Review by barneygale — APPROVED** [review]
Smashing, thanks so much

**Pull request merged**